### PR TITLE
added copy token button

### LINKED ISSUE #8: Add "copy to clipboard" button to the generated tokens in the webview
This would make it easier to use the generated tokens.

### PR DESCRIPTION
**Pull request by 0xkarambit**
added copy to clipboard button and changed readonly attributes to readOnly.
resolved #8

### CODE REVIEW

File: browser.jsx [hunk 1]
```diff
@@ -285,14 +290,14 @@ function UserTokenDisplay (props) {
...
-            Refresh token: <input type="text" readonly value={props.refreshToken} />
+            Refresh token: <input type="text" readOnly value={props.refreshToken} />
           </div>
           : <div>
             Refresh token: (None, you selected a temporary duration)
           </div>
       }
       <div>
-        Access token: <input type="text" readonly value={props.accessToken} />
+        Access token: <input type="text" readOnly value={props.accessToken} />
```
Comment (not-an-aardvark) [inline browser.jsx:326]: Would it make sense to have the `refreshToken` and `accessToken` copyable here too?

Comment (0xkarambit) [reply browser.jsx:326]: Yes, sorry i forgot about those !

**Comment by 0xkarambit** [comment]
Done :+1:

**Review by not-an-aardvark — APPROVED** [review]
Thanks!

**Comment by not-an-aardvark** [comment]
It looks like there are merge conflicts that need to be resolved since #21 was already merged.

**Comment by 0xkarambit** [comment]
Yup resolved !

**Pull request merged**